app/i18n/messages/sv.json

```diff
@@ -23,6 +23,7 @@
   "AboutPartners.aikaLogoAlt": "Logotypen för 6Aika",
   "AboutPartners.eakrLogoAlt": "Logotyp för Europeiska regionala utvecklingsfonden",
   "AboutPartners.euVipuvoimaaLogoAlt": "Hävkraft från EU-logo",
+  "SkipLink.text": "Gå direkt till innehållet",
```

Comment:
This property name is different between translation files

Reply:
Good catch, fixed 👍